examples/lessons_api_example.py

```diff
@@ -0,0 +1,113 @@
+# Run with Python 3
+import requests
+import pandas as pd
+import math
+
+
+class figure():
```

Comment:
pep8 refers to start all classnames with Capital, so this should be `class Figure()`